sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -19,11 +19,13 @@
 
     @Override
     protected void beforeTest() {
-        beforeTestSetup();
-        client = clientSetup(httpPipeline -> new TextAnalyticsClientBuilder()
-            .endpoint(getEndPoint())
-            .pipeline(httpPipeline)
-            .buildAsyncClient());
+//        beforeTestSetup();
```

Comment:
Can you run a test if you comment out these lines?

Reply:
My IDE isn't picking up the env variables correctly so for now doing it by using the subscription key manually.

Reply:
@mssfang  could you run it in live mode and transfer the test records on the main PR, till my IDE starts to pick up the vars